### PR TITLE
Stack - Select on which registries to push stack images

### PR DESCRIPTION
**Pull request by pacostas**
## Summary

This PR adds an extra option, which specifies on which registries the stack should push. Specifically, if on the root directory of the implementation, there is a file called `registries.json` with the below content
```
{
  "dockerhub": true,
  "GCR": false
}
``` 
it means that the action will push the stack images to `dockerhub` but NOT to `GCR`. 

In case the `registries.json` file does not exist, it fallbacks to the current behavior which is to push on both registries.

In case where an attribute is NOT specified, then the default value is `true`.

## Use Cases

This is useful for projects being under paketo-community organization, where GCR is not available.

## Checklist

* [x] I have viewed, signed, and submitted the Contributor License Agreement.
* [ ] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [x] I have added an integration test, if necessary.
* [x] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [x] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

**Comment by pacostas** [comment]
@mhdawson @ForestEckhardt

File: stack/.github/workflows/push-image.yml [hunk 1]
```diff
@@ -74,15 +101,24 @@ jobs:
         GCR_PASSWORD: ${{ secrets.GCR_PUSH_BOT_JSON_KEY }}
         GCR_PROJECT: "${{ github.repository_owner }}"
       run: |
+
+        if [ "${{ needs.preparation.outputs.push_to_dockerhub }}" == "true" ]; then
+        echo "${DOCKERHUB_PASSWORD}" | sudo skopeo login --username "${DOCKERHUB_USERNAME}" --password-stdin index.docker.io
```
Comment (robdimsdale) [inline stack/.github/workflows/push-image.yml:106]: Looks like this line is here multiple times. Am I missing something?

**Review by robdimsdale — COMMENTED** [review]
This looks good, with one minor question (inline) about potentially duplicated lines.

File: stack/.github/workflows/push-image.yml (hunk 1 above)
Comment (pacostas) [reply stack/.github/workflows/push-image.yml]: oops, I fixed it, the indentation was also wrong. I messed it up while committing it. Thanx for your time :) Feel free for any other suggestions, like the format of the registries.json file or if you think we should have that actions splitter into two one for the GCR and one for the dockerhub.

**Comment by pacostas** [comment]
@ForestEckhardt @robdimsdale  I think this PR is ready to be landed.

**Review by ForestEckhardt — APPROVED** [review]

**Pull request merged**